cg/meta/clean/flow_cell_run_directories.py

```diff
@@ -104,15 +103,41 @@ def archive_sample_sheet(self) -> None:
             return
         LOG.info("Sample sheet found!")
         hk_bundle: Bundle = self.hk.bundle(name=self.id)
-        hk_tags: List[str] = [SequencingFileTag.ARCHIVED_SAMPLE_SHEET, self.id]
+        hk_tags: List[str] = [self.id]
         if hk_bundle is None:
             LOG.info(f"Creating bundle with name {self.id}")
-            self.hk.create_new_bundle_and_version(name=self.id)
-        elif self.hk.get_file_from_latest_version(bundle_name=hk_bundle.name, tags=hk_tags):
-            LOG.warning("Sample sheet already included!")
-            return
+            hk_bundle = self.hk.create_new_bundle_and_version(name=self.id)
+        sample_sheets_from_latest_version: Optional[
+            List[File]
+        ] = self.get_sample_sheets_from_latest_version(hk_bundle_name=hk_bundle.name)
+        if sample_sheets_from_latest_version:
```

Comment:
Yeah, just saw that when I was switching to "samplesheet" instead of "archived_sample_sheet" since that's what @karlnyr suggested.

Reply:
Good catch! Not necessary at all but I got confused by the erroneous type hint. get_sample_sheets_from_latest_version will at worst return an empty list, not None. If it would return None, the if statement would be necessary.